### PR TITLE
Release as gem

### PR DESCRIPTION
**Pull request by neckhair**

### CODE REVIEW

**Comment by neckhair** [comment]
Waiting for https://github.com/johnewart/ruby-opentsdb/pull/2 to be merged.

**Comment by born4new** [comment]
It has been merged to master, but not yet released on rubygems.

**Comment by johnewart** [comment]
FYI: I released it today -- going forward releases should be automated via Travis

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,4 @@
 source 'https://rubygems.org'
 
-gem 'opentsdb', github: 'ninech/ruby-opentsdb'
```
Comment (born4new) [inline Gemfile]: Finally 🌈

**Pull request merged**